### DISCUSSION TITLE
How can I disable a control for the table but not for the docs?

### DISCUSSION THREAD

**Discussion by StephenSHorton** [storybookjs/storybook#29056] [Help] [answered]
### Summary

I have been made aware that I can use
```tsx
table: {
  disable: true,
},
```
To disable a control from the storybook. But this also disables it from the docs page using the tag `['autodocs']`. I want to still show props that are relevant for the component, but disable their control functionality.

**Comment by Marklb** [comment, marked as answer]
I think those tables are the same component and don't have much configuration based on where used. If you are just wanting to disable the control, use `control: false`.

https://storybook.js.org/docs/essentials/controls#disable-controls-for-specific-properties

```ts
{
  ...,
  argTypes: {
    myArg: {
      control: false,
    },
  },
}
```
